traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -650,10 +695,27 @@ func getServers(h http.Header, params map[string]string, tx *sqlx.Tx, user *auth
 		if userErr != nil || sysErr != nil {
 			return nil, 0, errors.New("Forbidden"), sysErr, http.StatusForbidden, nil
 		}
+
+		var joinSubQuery string
+		if version.Major >= 3 {
+			const hasRequiredCapabilitiesQuery = `
+SELECT EXISTS(
```

Comment:
Would you mind indenting this consistent with its containing block? It's not required by `go fmt` and it's fine syntactically as-is, but it breaks some text editors' code folding.

Reply:
Moved the query to the `deliveryservice` package in 51446fa30, which should also fix code folding.